src/pools/ID3MPool.sol

```diff
@@ -0,0 +1,120 @@
+// SPDX-License-Identifier: AGPL-3.0-or-later
```

Comment:
From Compound review - Add an interface to returnsthe address of what token anddress you get on exit if it exists, or address 0 if it doesn't exit.

Reply:
sorry @talbaneth I'm not sure i follow this. was the comment to return the address of `adai`/`cdai` from a new function in the interface?

Reply:
yes, so something like:
`
function redeemable() external pure override returns (address) {
   return address(adai);
}
`
Name of the function is up for debate.

Reply:
added 0c769b1